aws-single-page-static-site/README.md

```diff
@@ -0,0 +1,17 @@
+# Module template
```

Comment:
We should explain a bit about what this module is useful for? Like why is this specific to single page sites?